README.md

```diff
@@ -57,6 +57,9 @@ Get all your messages.
 Send a message with content `text`.
 Either specify an existing `chat_id`, or both `user` and `group_uid` for a new chat.
 
+### get_export()
+Get excel report for an event attendance, available via the web client.
```

Comment:
I think this should be something like "Get Excel attendance report for a single event..."

Reply:
Yeah i like that, more explicit, done!